src/Microsoft.DotNet.ServiceFabric.ServiceHost/ServiceHost.cs

```diff
@@ -231,16 +231,17 @@ private static void ConfigureDefaultServices(IServiceCollection services)
             services.AddOptions();
             services.SetupConfiguration();
             services.TryAddSingleton(InitializeEnvironment());
-            services.TryAddSingleton(b => (Microsoft.Extensions.Hosting.IHostingEnvironment) b.GetService<HostingEnvironment>());
-            services.TryAddSingleton(b => (Microsoft.AspNetCore.Hosting.IHostingEnvironment) b.GetService<HostingEnvironment>());
+
```

Comment:
What is this change?

Reply:
I think last week when I got the latest from master, I was not able to run it locally and you had a PR out to fix this error. I made the same changes in my local to make my code run. I committed this by mistake. This is the same as master.

Reply:
`git checkout master -- src/Microsoft.DotNet.ServiceFabric.ServiceHost/ServiceHost.cs` will get you back to the master version.